timeseries/tests/unittests/test_predictor.py

```diff
@@ -6,7 +6,7 @@
 import pandas as pd
 import pytest
 
-import autogluon.core as ag
+from autogluon.common.space import Bool, Categorical, Int, Real, Space
```

Comment:
I think the following code would still work and require fewer changes.
```python
import autogluon.common as ag

...
ag.space.Int(1, 3)
....
```

I personally also like `ag.space.Categorical` more than `Categorical` since this way it's always clear that this is a search space. This also would help to avoid potential clashes, e.g., with `torch.distributions.Categorical`.

Reply:
What you describe makes sense. We are trying to unify the usage of the search space in our code base and we think it's not a good practice to import the whole common module while all we are using are search spaces. Maybe a better alternative would be `from autogluon.common import space` then `space.XXX`. And when there are potential other search namespace, we can do `from autogluon.common import space as ag_space`

Reply:
I agree, this totally makes sense